### PR TITLE
Rewrite in ES6+

### PR DESCRIPTION
**Pull request by steelbrain**
Beep Beep

[Review on Reviewable](https://reviewable.io/reviews/atomlinter/atom-linter/67)

### CODE REVIEW

**Comment by steelbrain** [comment]
Alright guys, I am done. The specs are passing, all the functions have been ported. We now also keep dist (compiled) files in the repo, that change was necessary to allow this module to be used in `linter-eslint`. Other than that I've `npm ignored` the specs and the sources.

**Comment by kepler0** [comment]
:+1:

**Comment by benjaminRomano** [comment]
Reviewed 8 of 8 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/atomlinter/atom-linter/67)*

**Comment by steelbrain** [comment]
Beep Beep

File: package.json [hunk 1]
```diff
@@ -24,42 +25,7 @@
     "xregexp": "^2.0.0",
     "tmp": "latest"
   },
-  "coffeelintConfig": {
```
Comment (Arcanemagus) [inline package.json]: This really shouldn't have been removed until the specs were converted to pure JS as well.

**Pull request merged**